### PR TITLE
Add navbar animation

### PR DESCRIPTION
**Pull request by Bekiboo**
Navbar Animation from June 21st Live-Coding session

### CODE REVIEW

File: navbar-animation/src/routes/contact/+page.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// import type { PageLoad } from '../$types';
```
Comment (Ennoriel) [inline navbar-animation/src/routes/contact/+page.ts]: Can you delete this file?

File: navbar-animation/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# create-svelte
```
Comment (Ennoriel) [inline navbar-animation/README.md]: Can you edit the read-me to explain what this example shows?

**Review by Ennoriel — CHANGES_REQUESTED** [review]

File: navbar-animation/src/routes/contact/+page.ts (hunk 1 above)
Comment (Bekiboo) [reply navbar-animation/src/routes/contact/+page.ts]: Yup, it was a test to see how the animation reacted to a loading page. It is indeed obsolete now.

File: navbar-animation/README.md (hunk 2 above)
Comment (Bekiboo) [reply navbar-animation/README.md]: Done

**Pull request merged**